config/coordinator.default.json

```diff
@@ -13,9 +13,21 @@
     "UpdateIntervalMs": 2000,
     "Queries": [
       {
-        "Description": "Request duration",
+        "Description": "Average request duration",
         "Query": "rate(mattermost_http_request_duration_seconds_sum[1m])/rate(mattermost_http_request_duration_seconds_count[1m])",
-        "Threshold": 0.2,
+        "Threshold": 0.1,
+        "Alert": true
+      },
+      {
+        "Description": "Rate of 500 errors",
+        "Query": "sum(rate(mattermost_api_time_count{status_code=\"500\"}[1m]))",
```

Comment:
Would it be better to detect errors from nginx, which would mean checking client side errors than server side. I was of the mind that nginx can return a 502 in cases when a server is unable to establish a connection. In those cases, there won't be an API response, and the client would still get a response (albeit 502) within timeout.

Reply:
Good point. I admit that was added before I decided to integrate client metrics.  
I guess we can convert that one to something like:

```
sum(rate(loadtest_http_errors_total[1m]))
```

Reply:
Sounds perfect.